modules/labs-react/search-form/lib/SearchForm.tsx

```diff
@@ -405,52 +549,74 @@ export class SearchForm extends React.Component<SearchFormProps, SearchFormState
       onInputChange,
       autocompleteItems,
       initialValue,
-      searchTheme,
+      searchTheme = SearchTheme.Light,
       rightAlign,
       allowEmptyStringSearch = false,
       ...elemProps
     } = this.props;
 
+    const isCustom = typeof searchTheme === 'object';
+
     return (
-      <StyledSearchForm
+      <form
         role="search"
         action="."
-        rightAlign={rightAlign}
-        grow={grow}
         aria-labelledby={labelId}
-        isCollapsed={isCollapsed}
         onSubmit={this.handleSubmit}
-        showForm={this.state.showForm}
-        {...elemProps}
+        {...mergeStyles(
+          elemProps,
+          searchFormStencil({
+            grow,
+            rightAlign,
+            isCollapsed: isCollapsed ? 'true' : 'false',
+            showForm: this.state.showForm ? 'true' : 'false',
+            height: typeof height === 'number' ? px2rem(height) : height,
+            isHiddenSubmitSearchIcon: !!isCollapsed && !this.state.showForm,
+            isHiddenOpenSeachIcon: !isCollapsed || (!!isCollapsed && this.state.showForm),
+            searchTheme: typeof searchTheme === 'number' ? searchTheme : undefined,
+            searchInputBackground:
+              isCustom && searchTheme.background ? searchTheme.background : undefined,
+            searchInputBackgroundFocus:
+              isCustom && searchTheme.backgroundFocus ? searchTheme.backgroundFocus : undefined,
+            searchInputBackgroundHover:
+              isCustom && searchTheme.backgroundHover ? searchTheme.backgroundHover : undefined,
+            searchInputColor: isCustom && searchTheme.color ? searchTheme.color : undefined,
+            searchInputColorFocus:
+              isCustom && searchTheme.colorFocus ? searchTheme.colorFocus : undefined,
+            searchInputPlaceholderColor:
+              isCustom && searchTheme.placeholderColor ? searchTheme.placeholderColor : undefined,
+            searchInputPlaceholderColorFocus:
+              isCustom && searchTheme.placeholderColorFocus
+                ? searchTheme.placeholderColorFocus
+                : undefined,
+            searchInputBoxShadow:
+              isCustom && searchTheme.boxShadow ? searchTheme.boxShadow : undefined,
+            searchInputBoxShadowFocus:
+              isCustom && searchTheme.boxShadowFocus ? searchTheme.boxShadowFocus : undefined,
```

Comment:
I'm not a fan of so much glue logic in the render function. If the variable names matched the `searchTheme` object, we could remove it:

```tsx
const isCustom = !['light', 'dark', 'transparent'].includes(searchTheme);

...
searchTheme: !isCustom ? searchTheme : undefined,
{...(isCustom ? searchTheme : {})}
```